src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Methode completeOrder was called. Params: userId={}", userId);
```

Comment:
Let's make our log messages informative. The method name in the log message has a typo ('Methode' instead of 'Method') and it would be more informative to include the action being completed. For example: logger.debug("Method completeOrder was called to finalize the order. Params: userId={}", userId);